packages/wallets/src/components/WalletsActionScreen/WalletsActionScreen.tsx

```diff
@@ -30,26 +32,41 @@ const WalletsActionScreen: React.FC<PropsWithChildren<TProps>> = ({
     icon,
     onAction,
     title,
+    variant,
 }) => {
     const { isMobile } = useDevice();
 
     return (
         <div className='wallets-action-screen'>
-            <div className='wallets-action-screen__icon'>{icon}</div>
+            {icon && <div className='wallets-action-screen__icon'>{icon}</div>}
 
-            <div className='wallets-action-screen__title'>
-                <WalletText size={isMobile ? 'lg' : '2xl'} weight='bold'>
-                    {title}
-                </WalletText>
-            </div>
+            {title && (
+                <div className='wallets-action-screen__title'>
+                    <WalletText size={isMobile ? 'lg' : '2xl'} weight='bold'>
+                        {title}
+                    </WalletText>
+                </div>
+            )}
 
-            <div className='wallets-action-screen__subtitle'>
-                <WalletText size={isMobile ? 'md' : 'lg'}>{description}</WalletText>
-            </div>
+            {description && (
+                <div className='wallets-action-screen__subtitle'>
+                    <WalletText size={isMobile ? 'md' : 'lg'}>{description}</WalletText>
+                </div>
+            )}
 
-            <div className='wallets-action-screen__button'>
-                {actionText && <WalletButton color='primary' onClick={onAction} size='lg' text={actionText} />}
-            </div>
+            {actionText && onAction && (
```

Comment:
@lubega-deriv 
I'm still thinking about hiding `variant, onAction and actionText` props into a variable: `action`.
and i would like to add `disabled` prop too.
So basically we can add:
```
action : {
variant?: NonNullable<React.ComponentProps<typeof WalletButton>['variant']>;
onAction?: VoidFunction;
actionText?: string;
disabled?: boolean;
}
```

Reply:
Updated along with other components which uses `WalletsActionScreen` in the package 👍

Reply:
Merged with @farhan-nurzi-deriv's updated `WalletsActionScreen` and added the disabled property into it

Reply:
sure 👍